src/app/external-cluster-wizard/steps/summary/external-cluster-details/external-cluster-details-aks/template.html

```diff
@@ -0,0 +1,83 @@
+<!--
+Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<km-external-cluster-details [cluster]="cluster">
+  <ng-container leftColumn>
+    <km-property>
+      <div key>Location</div>
+      <div value>
+        <span>{{ spec.location }}</span>
+      </div>
+    </km-property>
+
+    <km-property>
+      <div key>Resource Group</div>
+      <div value>
+        <span>{{ aks.resourceGroup }}</span>
+      </div>
+    </km-property>
+  </ng-container>
+
+  <ng-container rightColumn>
+    <div class="section-header">Node Pool Setting</div>
+    <km-property>
+      <div key>Node Pool Name</div>
+      <div value>
+        <span>{{ spec.machineDeploymentSpec.name }}</span>
+      </div>
+    </km-property>
+
+    <ng-container *ngIf="spec.machineDeploymentSpec.basicSettings as setting">
+      <km-property>
+        <div key>VM Size</div>
+        <div value>
+          <span>{{ setting.vmSize }}</span>
+        </div>
+      </km-property>
+
+      <km-property>
+        <div key>Count</div>
+        <div value>
+          <span>{{ setting.count }}</span>
+        </div>
+      </km-property>
+
+      <km-property>
+        <div key>Mode</div>
+        <div value>
+          <span>{{ setting.mode }}</span>
+        </div>
+      </km-property>
+
+      <km-property-boolean label="AutoScaling"
```

Comment:
i think maybe we need to show `autoScaling` always but if it's false then don't show the max and min propertes because now we r having error on disable `autoScaling`
![Screenshot from 2022-09-07 14-07-44](https://user-images.githubusercontent.com/85109141/188865118-92ec689f-5365-4550-a1f5-cedb4ff7ad03.png)